### ISSUE TITLE
Highways controller should have tests

### ISSUE THREAD

**Issue by joshproehl** [PSU-Capstone-2014-B/routelandia-server#6]
Bad josh, writing code without tests... Get on it!

**Comment by joshproehl** [comment]
Stubbed in joshproehl/bdd
Blocked by #33

**Comment by severity** [comment]
@joshproehl has written some tests for the highways controller, i'm not sure if we need more tests at this time or if we should wait until we think we are done to close this issue.

**Comment by severity** [comment]
Once the creating tests branch is added to master this issue can be closed :)

**Comment by severity** [comment]
The testing branch has been merged into master, closing this issue now :)
